examples/counterAndGauge.js

```diff
@@ -0,0 +1,37 @@
+// Increment a counter every 10ms
+var Prometheus = require("prometheus-client");
+
+var client = new Prometheus();
+
+var counter = client.newCounter({
+    namespace: "counter_test",
+    name: "elapsed_counters_total",
+    help: "The number of counter intervals that have elapsed."
+});
+
+gauge = client.newGauge({
+    namespace: "counter_test",
+    name: "random_number",
+    help: "A random number we occasionally set."
+});
+
+setInterval(function() {
+    counter.increment({
+        period: "1sec" //period is a custom label name in this case with a value of "1sec"
+    });
+}, 1000);
+
+setInterval(function() {
+    counter.increment({
+        period: "2sec" //period label now has a value of "2sec"
```

Comment:
Might want to tweak the comment here... We're not changing the label on the previous series; we're creating a new series with a period label of `2sec`. Both will be tracked.